### PR TITLE
/pricing/infra => /pricing/pro

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- moved /pricing/infra to /pricing/pro
- added a redirect for the above
- updated the copy on /pricing/pro according to the copy doc: https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit#heading=h.l83dv5879ajz

## QA

- Visit https://ubuntu-com-12083.demos.haus/pricing/infra
- See that you are redirected to https://ubuntu-com-12083.demos.haus/pricing/pro
- See that the page matches the copy doc

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6016

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
Can we please align tables as per the [modular table guidelines ](https://discourse.ubuntu.com/t/table-cell-contents/19858?u=lyubomir-popov)

**Comment by lechsandecki** [comment]
@sowasred2012 Expanded Security Maintenance for Infrastructure, the "for Infrastructure" is missing

File: templates/shared/pricing/_ua-for-infrastructure.html [hunk 1]
```diff
@@ -1,171 +1,88 @@
...
-          <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
-          <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
-          <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
+          <td><a href="/security/certifications">FIPS 140</a> certified crypto modules</td>
+          <td class="u-align--right">Included</td>
+          <td class="u-align--right">Included</td>
+          <td></td>
         </tr>
         <tr>
           <td><a href="/security/certifications#common-criteria">Common Criteria EAL2</a></td>
```
Comment (minkyngkm) [inline templates/shared/pricing/_ua-for-infrastructure.html:76]:
```suggestion
          <td><a href="/security/certifications#common-criteria">Common Criteria EAL2 certified</a></td>
```

File: templates/shared/pricing/_ua-for-infrastructure.html [hunk 2]
```diff
@@ -1,171 +1,88 @@
...
-          <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
-          <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
-          <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
+          <td class="u-align--right">Included</td>
+          <td class="u-align--right">Included</td>
+          <td></td>
         </tr>
         <tr>
           <td><a href="/security/certifications#cis">CIS Benchmark</a></td>
```
Comment (minkyngkm) [inline templates/shared/pricing/_ua-for-infrastructure.html:82]:
```suggestion
          <td>Ubuntu Security Guide (USG) with <a href="/security/certifications#cis">CIS Benchmark</a>and DISA-STIG profiles</td>
```

**Review by minkyngkm — COMMENTED** [review]
Looking good @sowasred2012. I just left some comments for minor things and I'm wondering if we are using "Included"/"Not included" instead of this icon as on [the current page](https://ubuntu.com/pricing/infra) the icon is used.
image

**Comment by sowasred2012** [comment]
@minkyngkm nice spots - and yes, after conversation with @lyubomir-popov about table column alignment, we went with text rather than icons.

File: templates/shared/pricing/_ua-for-infrastructure.html [hunk 3]
```diff
@@ -1,171 +1,88 @@
...
+          <td class="u-align--right">Included</td>
+          <td class="u-align--right">Included</td>
+          <td></td>
         </tr>
         <tr>
-          <td>Ceph/Swift raw storage support included</td>
-          <td class="u-align--center">&ndash;</td>
-          <td class="u-align--center">192TB per storage node</td>
-          <td class="u-align--center">192TB per storage node</td>
+          <td>Ubuntu Security Guide (USG) with <a href="/security/certifications#cis">CIS Benchmark</a>and DISA-STIG profiles</td>
```
Comment (minkyngkm) [inline templates/shared/pricing/_ua-for-infrastructure.html:82]:
```suggestion
          <td>Ubuntu Security Guide (USG) with <a href="/security/certifications#cis">CIS Benchmark</a> and DISA-STIG profiles</td>
```

**Review by minkyngkm — APPROVED** [review]
Just need a single space after a tag in line 82. Other than that LGTM! 👍

**Pull request merged**